ibm/mas_devops/roles/kafka/tasks/main.yaml

```diff
@@ -6,4 +6,4 @@
       - "Kafka Action .................... {{ kafka_action }}"
 
 - name: Provision or Deprovision Kafka Instance
-  include_tasks: tasks/provider/{{ kafka_provider }}/{{ kafka_action }}.yml
+  include_tasks: ../roles/kafka/tasks/provider/{{ kafka_provider }}/{{ kafka_action }}.yml
```

Comment:
This shouldn't be needed. What was being run when you saw this problem? It sounds like the role path was not set correctly?

Reply:
I was testing "mas update" and it consistently fails with an invalid path that includes the directory "playbooks". Thus this change prepends "../roles/kafka/".

So there must be an problem with the role path for update in the cli?

Reply:
@racree Was this running inside the cli docker container or just running `mas update` directly on your machine? We don't seem to need this change in other places so I suspect something might be changing the path before this.

Reply:
I ran the following steps inside the cli docker container.  
Steps:

1. mas install
2. installed kafka using "ROLE_NAME=kafka ansible-playbook ibm.mas_devops.run_role"
3. mas update

mas update consistently failed with a file not found.

Reply:
the failure happens in the tekton pipeline, so don't think this environmental